.travis.yml

```diff
@@ -1,18 +1,29 @@
 language: python
 
+sudo: "required"
+
 python:
-  - "2.7"
-  - "3.5"
+    - "2.7"
+    - "3.5"
 
-sudo: "required"
+env:
+  global:
+    - DISPLAY=:99.0
 
 before_install:
-  - "pip install virtualenv"
-  - "virtualenv venv"
-  - "source venv/bin/activate"
+  - pip install mozdownload mozinstall
```

Comment:
These won't work with python 3.5 so either remove python 3.5 (and re-introduce it in another patch), or use a different method for downloading/installing Firefox.